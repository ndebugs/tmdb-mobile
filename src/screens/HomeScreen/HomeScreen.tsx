import React from 'react';
import {
  View,
  Text,
} from 'react-native';
import { Container, Header } from '../../components';

const HomeScreen = () => {
  return (
    <Container>
      <Header title="Home" />
      <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
        <Text testID="blankLabel">This is Home screen.</Text>
      </View>
    </Container>
  );
};

export default HomeScreen;
