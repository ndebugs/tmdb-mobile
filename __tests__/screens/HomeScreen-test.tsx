import React from 'react';
import { shallow } from 'enzyme';
import HomeScreen from '../../src/screens/HomeScreen';

describe('<HomeScreen>', () => {
  it('should match snapshot', () => {
    const component = shallow(<HomeScreen />);
    expect(component).toMatchSnapshot();
  });
});
