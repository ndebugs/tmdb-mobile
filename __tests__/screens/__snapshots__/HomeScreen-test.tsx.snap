// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeScreen> should match snapshot 1`] = `ShallowWrapper {}`;
